cloudsync/oauth/redir_server.py

```diff
@@ -156,6 +156,14 @@ def shutdown(self):
             self.__on_failure = None
         self.__thread = None
 
+    def server_close(self):
+        """Closes the server and joins all threads"""
+        if self.__api_server and not self.__running:
```

Comment:
If `not self.__running`, is the best behavior is to silently ignore a call to `server_close` ?

Reply:
moved discussion to VFM-10044